tests/integration/targets/iso_customize/files/grub.cfg

```diff
@@ -0,0 +1,35 @@
+set timeout=30
```

Comment:
You need an `aliases file in `tests/integration/targets/iso_customize/` for tests to actually run in CI.

Reply:
Thanks. Done